### DISCUSSION TITLE
Any verified tft displays?

### DISCUSSION THREAD

**Discussion by fridgetheorem** [fooandbar/Mks-Robin-Nano-Marlin2.0-Firmware-For-Selpic-Star-A.LABISTS-SX1#14] [Q&A]
I am looking to install this firmware on my Labists SX1, but I don't have a 32-pin TFT display. Given that Selpic no longer sells the display, are there any verified working displays out there still?

**Comment by fooandbar** [comment]
Hi,
You can use MKS TFT35 with 32 pin FFC.
https://www.aliexpress.com/item/1005002898776913.html
